src/server/config.js

```diff
@@ -58,3 +58,10 @@ function getValue(prop, optional) {
     }
     return config[prop];
 }
+
+function newInstance() {
+    config = {};
+    simulationFilePath = null;
```

Comment:
Yes, this branch is out of date, for a week I guess, I will update it today :) in a couple of minutes. Thanks @dlebu !